selfdrive/car/toyota/interface.py

```diff
@@ -254,7 +255,7 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), has_relay=False,
 
     elif candidate == CAR.COROLLA_TSS2:
       stop_and_go = True
-      ret.safetyParam = 55
+      ret.safetyParam = 53
```

Comment:
need separate dbc.

Reply:
![Screen Shot 2020-12-22 at 11 31 55 AM](https://user-images.githubusercontent.com/67883352/102926116-5720c980-4449-11eb-9d3c-1025e90a080a.png)

Reply:
I think changing it for all is also ok.